doc_source/faces-detect-images.md

```diff
@@ -350,6 +350,92 @@ You can provide the input image as an image byte array \(base64\-encoded image b
       });
    ```
 
+------
+#### [ GO ]
+
+   This example displays the estimated age range , position , emotion and Gender for each detected faces\. Change the value of `photo` to the image file name\. Change the value of `bucket` to the Amazon S3 bucket where the image is stored\.
+
+   ```
+   //Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
All of our code examples use the following copyright:

// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX - License - Identifier: Apache - 2.0

Reply:
Hi Doug,
I have updated the copyright statement.